specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-11-30-preview/definitions/MigrateMySqlAzureDbForMySqlOfflineTask.json

```diff
@@ -384,6 +441,13 @@
           "additionalProperties": {
             "type": "string"
           }
+        },
+        "tablesToMigrateSchema": {
+          "type": "object",
+          "description": "Mapping of source to target tables for schema migration",
+          "additionalProperties": {
+            "type": "string"
```

Comment:
Can we also add the selected non table objects here?
selectedViews, selectedTriggers, selectedRoutines, selectedEvents

Reply:
Added. We do not set these attributes from the UI, so I missed it.